x-pack/plugins/threat_intelligence/cypress/integration/sources/indicators.spec.ts

```diff
@@ -13,24 +13,53 @@ import {
   FLYOUT_TITLE,
   INDICATORS_TABLE,
   TOGGLE_FLYOUT_BUTTON,
+  FILTERS_GLOBAL_CONTAINER,
+  TIME_RANGE_PICKER,
+  QUERY_INPUT,
+  TABLE_CONTROLS,
+  INDICATOR_TYPE_CELL,
+  EMPTY_STATE,
 } from '../../screens/indicators';
 import { login } from '../../tasks/login';
 
 before(() => {
   login();
 });
 
-describe('Indicators page', () => {
+/**
+ * Time range extended to 15 years back to ensure fixtures are showing up correctly
+ */
+const THREAT_INTELLIGENCE_15Y_DATA =
+  '/app/security/threat_intelligence?indicators=(filterQuery:(language:kuery,query:%27%27),filters:!(),timeRange:(from:now-15y/d,to:now))';
+
+describe('Indicators page basics', () => {
   before(() => {
-    cy.visit('/app/security/threat_intelligence');
+    cy.visit(THREAT_INTELLIGENCE_15Y_DATA);
   });
 
-  it('should navigate to the indicators page, click on a flyout button and inspect flyout', () => {
+  it('should render the basic page elements', () => {
     cy.get(DEFAULT_LAYOUT_TITLE).should('have.text', 'Indicators');
 
     cy.get(INDICATORS_TABLE).should('exist');
 
-    cy.get(TOGGLE_FLYOUT_BUTTON).should('exist').first().click();
+    cy.get(FILTERS_GLOBAL_CONTAINER).should('exist');
+
+    cy.get(`${FILTERS_GLOBAL_CONTAINER} ${TIME_RANGE_PICKER}`).should('exist');
+  });
+});
+
+describe('Indicator page flyout', () => {
+  before(() => {
+    cy.visit(THREAT_INTELLIGENCE_15Y_DATA);
+  });
+
+  it('should show the indicator flyout on ioc click', () => {
+    cy.get(TABLE_CONTROLS).should('contain.text', 'Showing 1-25 of');
```

Comment:
do we really want to test the table controls here? Don't get me wrong it's useful but for me e2e tests are more about testing functionality, not necessarily the fact that we decided to default to 25 items per page...
Plus this is something that comes from the `eui` library, if for some reason they decided to change the verbiage to for example just `1 to 25 of` this test would fail even though our logic is still correct

Reply:
It's just to test it the data load is complete. I had false positives with other elements :/.